### DISCUSSION TITLE
Transient state

### DISCUSSION THREAD

**Discussion by 0xair** [foundry-rs/foundry#8655] [Help]
Hello. In a contract, there is a modifier that checks which address is stored in a specific transient slot (tstore). I'm trying to test this contract. Is there any way in Foundry tests to bypass this check? MockCall doesn't help because the check is internal. I need something like vm.store, but for transient. Alternatively, how can I bypass a single revert() in the modifier?

**Comment by zerosnacks** [comment]
Hi @0xair I think this will be possible once https://github.com/foundry-rs/foundry/issues/8165 has been implemented

Let me know if this is still relevant to you
